lemming.go

```diff
@@ -87,6 +106,60 @@ func WithDeviceConfig(c []byte) Option {
 	}
 }
 
+// WithGRIBIAddr is a device option that specifies that the gRIBI address.
+func WithGRIBIAddr(host string, port int) Option {
```

Comment:
it seems like we could just be using a string everywhere, there no reason to separate host and port, unless i'm missing something

Reply:
Done